src/main/java/news/caughtup/database/MySQLDriver.java

```diff
@@ -62,7 +62,7 @@ public MySQLDriver() {
 		if (sqlProps == null) {
 			sqlProps = new Properties();
 			FileInputStream in;
-			String sqlPropsPath = System.getProperty( "catalina.base" ) + "/webapps/caughtup/WEB-INF/classes/SQLQueries.properties";
+			String sqlPropsPath = System.getProperty( "catalina.base" ) + "/wtpwebapps/CaughtUp-Server/WEB-INF/classes/SQLQueries.properties";
```

Comment:
Just do `git update-index --assume-unchanged src/main/java/news/caughtup/database/MySQLDriver.java` and it won't commit changes. Although you have to change it back and push and then do that.